packages/pebble-shared/svgs/print.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
pebble icons should be 1000x1000 with no space btw in horizontal direction (left/right edges)